### ISSUE TITLE
Testing Privacy Settings Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4645]
# Page to Test:

Privacy Settings - https://openlibrary.org/account/privacy

# Steps

Steps for this task are outlined in #4521

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly No images found. We can close this issue.
